### ISSUE TITLE
Failed dependency for version 20191028.2 

### ISSUE THREAD

**Issue by qvdm** [albertogeniola/meross-homeassistant#23]
The manifest for version 20191028.2 specifies meross_iot==0.3.1.7, but the latest version of that library is 0.3.1.6, causing the installation to fail.

**Comment by albertogeniola** [comment]
Should be fixed now. There was a pipeline error while building the underlying version of the merossiot library.
Could you please confirm?

**Comment by luistaboada** [comment]
Had the same problem yesterday, seems to be working fine now. Thanks!

**Comment by qvdm** [comment]
Thanks for the quick fix.  

Not sure if a separate Issue needs to be opened, but as part of recovering from the problem yesterday, I tried downgrading the Meross component to the previous version.  The downgrade worked, but required a HA restart of course.  

Unfortunately the error condition was not cleared during the downgrade, causing HA to refuse to restart through the GUI.  That was fixed quickly by manually restarting, but at the time it was quite frustrating to have to figure out yet another issue.
